hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/command/AlterHoodieTableChangeColumnCommand.scala

```diff
@@ -102,4 +104,8 @@ case class AlterHoodieTableChangeColumnCommand(
         ", origin table schema :" + tableSchema + ", base path :" + metaClient.getBasePath)
     }
   }
+
+  // Add the comment to a column, if comment is empty, return the original column.
+  private def addComment(column: StructField, comment: Option[String]): StructField =
```

Comment:
addComment -> columnWithComment

Can we add a test case for it?

Reply:
> addComment -> columnWithComment

I just copyed from spark https://github.com/apache/spark/blob/67abc430140558e60c785b158e9199dc884fb15c/sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala#L390

Reply:
Originally, without this PR, the nullable attribute of the field might be changed after `alter table change column commnt`(not null -> nullable), but currently the spark field is always nullable, so no suitable test case has been found so far...

Reply:
Suggest that not only sync the new schema (with new comment) to HMS, but also keep it in the hudi own meta data. Then you can add a UT by check whether the underlying avro schema is what you expect after changing the comment.

Reply:
> but also keep it in the hudi own meta data

Guess we already did that for purpose of schema evolution, like the `InternalSchema` thing.